HPXMLtoOpenStudio/resources/geometry.rb

```diff
@@ -445,7 +445,7 @@ def self.getSurfaceZValues(surfaceArray)
     zValueArray = []
     surfaceArray.each do |surface|
       surface.vertices.each do |vertex|
-        zValueArray << UnitConversions.convert(vertex.z, 'm', 'ft')
+        zValueArray << UnitConversions.convert(vertex.z, 'm', 'ft').round(5)
```

Comment:
All the changes here and below to the HPXMLtoOpenStudio measure are from pulling in master. (We can't pull master into the base `build-res-hpxml-v3` branch because tests would fail.)